api/src/components/coauthor/controller.ts

```diff
@@ -253,10 +253,13 @@ export const updateConfirmation = async (
             });
         }
 
-        // Is the publication in draft?
-        if (publication.currentStatus === 'LIVE') {
+        // Is the publication in locked mode?
+        if (publication.currentStatus !== 'LOCKED') {
             return response.json(403, {
-                message: 'This publication is LIVE and therefore cannot be edited.'
+                message:
+                    publication.currentStatus === 'LIVE'
+                        ? 'This publication is LIVE and therefore cannot be edited.'
+                        : 'This publication is in DRAFT and therefore cannot be edited.'
```

Comment:
Error messages are not related to what this controller is actually doing, the old error message was wrong also. Through this controller we're not editing the publication.

If currentStatus is DRAFT we can say something like: 'This publication is not ready for approval yet'
If it's LIVE, we can say something like: 'You cannot approve a LIVE publication'